includes/platform-checkout-user/class-platform-checkout-save-user.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Class PlatformCheckoutSaveUser
+ *
+ * @package WooCommerce\Payments\PlatformCheckout
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that adds a section to save new user for platform checkout on the frontend.
+ */
+class Platform_Checkout_Save_User {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action( 'wp_enqueue_scripts', [ $this, 'register_checkout_page_scripts' ] );
+	}
+
+	/**
+	 * Load scripts and styles for checkout page.
+	 */
+	public function register_checkout_page_scripts() {
+		$script_src_url    = plugins_url( 'dist/platform-checkout.js', WCPAY_PLUGIN_FILE );
+		$style_url         = plugins_url( 'dist/platform-checkout.css', WCPAY_PLUGIN_FILE );
+		$script_asset_path = WCPAY_ABSPATH . 'dist/platform-checkout.asset.php';
+		$script_asset      = file_exists( $script_asset_path ) ? require_once $script_asset_path : [ 'dependencies' => [] ];
+
+		wp_register_style(
+			'WCPAY_PLATFORM_CHECKOUT',
+			$style_url,
+			[ 'wp-components' ],
```

Comment:
If wc-component is what we used, then this dependency will be different as well.